src/pages/index.tsx

```diff
@@ -129,6 +132,31 @@ const Home = ({
               </Paragraph>
             )}
 
+            {hasAccessToNewBichard && (
+              <>
+                <br />
+                <Link
+                  href="/bichard"
+                  basePath={true}
+                  className="govuk-button govuk-button--start govuk-!-margin-top-5"
+                  id="bichard-link"
+                >
+                  {"Access New Bichard"}
+                  <svg
+                    className="govuk-button__start-icon"
+                    xmlns="http://www.w3.org/2000/svg"
+                    width="17.5"
+                    height="19"
+                    viewBox="0 0 33 40"
+                    aria-hidden="true"
+                    focusable="false"
+                  >
+                    <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z" />
+                  </svg>
+                </Link>
+              </>
```

Comment:
I'm assuming this button/SVG is a copy-paste of the same button above that's used to access old Bichard? Perhaps we should pull it out into a component?

Reply:
Ok, I think we were under the impression that this button will eventually replace the old but 100% can break it out into it own component